echo: add exit code command (#10538)
tldr-pages/tldr: pages/common/echo.md

- Enable interpretation of backslash escapes (special characters):

`echo -e "{{Column 1\tColumn 2}}"`

> - Print success or failure message of the last executed command:

Reviewer: ```suggestion
- Print the exit status of the last executed command (Note: In Windows Command Prompt, the equivalent command is `echo %errorlevel%`):
```

> - Print success or failure message of a recently executed command:

Reviewer 2: ```suggestion
- Print success or failure message of the last executed command:
```
